src/main/scala/zio/s3/package.scala

```diff
@@ -130,7 +132,8 @@ package object s3 {
         bucketName: String,
         key: String,
         contentType: String,
-        content: ZStream[R, Throwable, Byte]
+        content: ZStream[R, Throwable, Byte],
+        metadata: Map[String, String]
```

Comment:
a default value would be great
`metadata: Map[String, String] = Map.empty`

Reply:
https://github.com/zio/zio-s3/commit/6a023d344d075aee21373730e85043bb71048b41